.github/workflows/release.yaml

```diff
@@ -0,0 +1,35 @@
+name: "tagged-release"
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  tagged-release:
+    name: "Tagged Release"
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up golang
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.x
+      - name: install dependencies
+        run: |
+          sudo apt-get update
```

Comment:
Is there some way to reference all this code so we don't have two copies of the same thing? Or perhaps only run the automatic releases bit with a condition?

Reply:
Your first solution seems reasonable to me.